mgnify_pipelines_toolkit/analysis/amplicon/amplicon_utils.py

```diff
@@ -50,15 +64,14 @@ def get_read_count(read_path, type='fastq'):
             '^>',
             read_path
         ]
-
-    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    stdout, stderr = proc.communicate()
+        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        stdout, stderr = proc.communicate()
 
     read_count = int(stdout.strip())
```

Comment:
I would add a sanity check here with `stdout.isdigit()`, just in case. Docs https://docs.python.org/3/library/stdtypes.html#str.isdigit